proto/cosmos/tx/signing/signing.proto

```diff
@@ -20,3 +22,49 @@ enum SignMode {
     // Amino JSON and will be removed in the future
     SIGN_MODE_LEGACY_AMINO_JSON = 127;
 }
+
+// SignatureDescriptors wraps multiple SignatureDescriptor's.
+message SignatureDescriptors {
+    // signatures are the signature descriptors
+    repeated SignatureDescriptor signatures = 1;
+}
+
+// SignatureDescriptor is a convenience type which represents the full data for a
+// signature including the public key of the signer, signing modes and the signature
+// itself. It is primarily used for coordinating signatures between clients.
+message SignatureDescriptor {
```

Comment:
This is super similar to SignatureV2. Could we remove SignatureV2 and use this one everywhere?

Reply:
That's fine we can